kura/distrib/src/main/resources/unfiltered/deb/control/postinst

```diff
@@ -82,6 +82,10 @@ function postInstall {
     
     echo ""
     echo "Finished. KURA has been installed to ${INSTALL_DIR}/kura and will start automatically after a reboot"
+    echo ""
+    echo "****************************************************************"
+    echo "WARNING: Set the crypto service encryption key as explained in Eclipse Kura documentation before rebooting to have snapshot encryption enabled."
```

Comment:
Correct me if I'm wrong but snapshot encryption is still available even if no encryption key is provided given that we use the default (albeit well known) one.

I would rephrase this as:


```suggestion
    echo "****************************************************************"
    echo "WARNING: Set the crypto service encryption key as explained in Eclipse Kura documentation before rebooting for additional security."
```

but feel free to disagree